clients/lsp-rust.el

```diff
@@ -342,6 +342,13 @@ PARAMS progress report notification data."
   :group 'lsp-rust-analyzer
   :package-version '(lsp-mode . "6.2"))
 
+(defcustom lsp-rust-analyzer-server-format-inlay-hints t
```

Comment:
in general I'd prefer if the defcustoms mirror the naming of the options in rust-analyzer, but I recognize it's too late here :/

Reply:
Left that on the wrong line, it was referring to `lsp-rust-analyzer-display-lifetime-elision-hints` etc.